File: Iniciante/URI 1041.lua
line = {}
n = 0
for i in string.gmatch(io.read("*l"), "%S+") do
   n = n + 1
   line[n] = i
end
x = tonumber(line[1])
y = tonumber(line[2])

if (x == 0 and y == 0) then
    print('Origem')
elseif (x == 0) then
    print('Eixo Y')
elseif (y == 0) then
    print('Eixo X')
elseif (y > 0 and x > 0) then
    print('Q1')
elseif (y > 0 and x < 0) then
    print('Q2')
elseif (y < 0 and x < 0) then
    print('Q3')
elseif (y < 0 and x > 0) then
    print('Q4')
end

File: Iniciante/URI 1041.clj
(require '[clojure.string :as str])

(def line (str/split (read-line) #" "))
(def x (Double/parseDouble (get line 0)))
(def y (Double/parseDouble (get line 1)))

(cond
    (and (== x 0) (== y 0)) 
        (println "Origem")
    
    (== x 0) 
        (println "Eixo Y")
    
    (== y 0) 
        (println "Eixo X")
    
    (and (> y 0) (> x 0)) 
        (println "Q1")

    (and (> y 0) (< x 0)) 
        (println "Q2")

    (and (< y 0) (< x 0)) 
        (println "Q3")

    (and (< y 0) (> x 0)) 
        (println "Q4")
)